### PR TITLE
Own pings - export

### PR DESCRIPTION
**Pull request by cbart**
Part of #52578

This pull requests adds a single count to pings that stores the number of assigned owners for Own.

## Test plan

Extend existing serialization unit tests as well as usagestats & store db tests.

### CODE REVIEW

File: internal/database/assigned_owners.go [hunk 1]
```diff
@@ -102,6 +104,17 @@ func (s assignedOwnersStore) DeleteOwner(ctx context.Context, assignedOwnerID in
...
 
+const countAssignedOwnersFmtstr = `SELECT COUNT(*) FROM assigned_owners`
+
+func (s assignedOwnersStore) CountAssignedOwners(ctx context.Context) (int32, error) {
+	r := s.QueryRow(ctx, sqlf.Sprintf(countAssignedOwnersFmtstr))
+	var count int32
+	if err := r.Scan(&count); err != nil {
+		return 0, err
+	}
+	return count, nil
```
Comment (sashaostrikov) [inline internal/database/assigned_owners.go:115]: nit: we can use `basestore`
```suggestion
	count, _, err := basestore.ScanFirstInt(s.db.Query(ctx, sqlf.Sprintf(countAssignedOwnersFmtstr)))
	if err != nil {
		return 0, err
	}
	return int32(count), err
```

File: internal/types/types.go [hunk 2]
```diff
@@ -1836,6 +1836,11 @@ type OwnershipUsageStatistics struct {
 
 	// Opening ownership panel.
 	OwnershipPanelOpened *OwnershipUsageStatisticsActiveUsers `json:"ownership_panel_opened,omitempty"`
+
+	// AssignedOwnersCount is the sum total of assigned owners. For instance
```
Comment (sashaostrikov) [inline internal/types/types.go:1840]: maybe? 
```suggestion
	// AssignedOwnersCount is the total number of assigned owners. For instance
```

**Review by sashaostrikov — APPROVED** [review]
LGTM! Added a couple minor comments.

**Review by nathan-downs — APPROVED** [review]
Saw some of the test payloads come across and used those to test against the schema changes. We're good to go on the data side.

**Review by coury-clark — APPROVED** [review]

**Pull request merged**